scripts/main/album.js

```diff
@@ -852,6 +890,22 @@ album.setPublic = function (albumID, e) {
 	});
 };
 
+album.setNSFW = function (albumID, e) {
```

Comment:
Would `toggleNSFW` fit better than `setNSFW` here?

Reply:
I would argue that toggle is more of an on-off temporary (a bit like the `h` short cut) while set is more in the length. Also that Set is more in line with the name of all the previously defined functions. :)

Reply:
In my mind "set" is more saving a specific value, whereas this is changing it, *then* saving it.